github/github_test.go

```diff
@@ -800,7 +800,7 @@ func TestCheckResponse(t *testing.T) {
 			CreatedAt *Timestamp `json:"created_at,omitempty"`
 		}{
 			Reason:    "dmca",
-			CreatedAt: &Timestamp{time.Date(2016, 3, 17, 15, 39, 46, 0, time.UTC)},
+			CreatedAt: &Timestamp{time.Date(2016, time.July, 17, 15, 39, 46, 0, time.UTC)},
```

Comment:
s/time.July/time.March/

I won't comment on any others... please make sure all the tests pass.
See the `CONTRIBUTING.md` file in the top of the repo for more details.

Thank you!

Reply:
Oh, such a silly mistake. Sorry for that :(